lib/lotus/helpers/numbers/number_formatter.rb

```diff
@@ -0,0 +1,52 @@
+module Lotus
+  module Helpers
+    module Numbers
+      # Numbers
+      #
+      # @since x.x.x
+      class NumberFormatter
+        # NumberFormatter
+        #
+        # @since x.x.x
+        # @api private
+
+        attr_accessor :number, :delimiter, :separator, :precision
```

Comment:
@jodosha I can make make `initialize` and the `attr_accessors` private but the `parse` method needs to be public. Is that what you mean?